### ISSUE TITLE
Atualização do livro!? (placas, componentes etc)

### ISSUE THREAD

**Issue by joaonicacio** [Patola/ebook#11]
Uau!

Gostei muitíssimo do material. Está de parabéns. Sinceramente uso como bíblia (cheguei a imprimir);

Dito isso, questiono se teria interesse em atualizar, mais precisamente quanto a placas, firmwares etc.

Mesmo assim fica meus parabéns pelo material.

**Comment by Patola** [comment]
Tenho interesse em atualizar sim, com certeza, João. No entanto minha vida mudou, mudei de emprego, e não tenho mexido mais diretamente com impressoras 3D, embora tenha uma no meu apartamento aqui na Polônia. Pretendo retomar alguma coisa do livro, mas não tenho prazo. Se você tiver interesse em mudar qualquer parte, é só mandar um pull request aqui no github que eu avalio e se for o caso, aprovo. Que tal?

**Comment by joaonicacio** [comment]
Ficaria honrado. Confesso que não era a minha ideia, nem tenho tempo livre também. Mexo com impressão por hobby, mesmo sabendo que não é bem um hobby.
Vou tentar coletar algo e quando tiver algo minimamente consolidado faço o pull request.

**Comment by joaonicacio** [comment]
Posso convidar outros da comunidade para fazer o mesmo?

**Comment by Patola** [comment]
Claro! Por favor, deve!! O livro foi feito pra ter contribuições comunitárias, mas o povo parece meio acanhado com isso. :P

**Comment by RafaelEstevamReis** [comment]
Closed ?

**Comment by farribeiro** [comment]
Acho que não... precisa de novos contribuidores

**Comment by Patola** [comment]
É, melhor deixar aberto por enquanto

**Comment by joaonicacio** [comment]
Por favor.

Também ando sem tempo, mas faço questão de ajudar da forma que for possível.
Adquiri inclusive uma Impressora estereolitográfica.

Vou fazer uma publicação no grupo de facebook. A maioria dos makers poderão se interessar em adquirir, e alguns outros, que tenham conhecimento, poderiam ajudar.
